composer.json

```diff
@@ -62,6 +62,16 @@
         "adhocore/jwt": "1.1.2",
         "slickdeals/statsd": "3.0.2"
     },
+    "repositories": [
+        {
+            "type": "git",
+            "url": "https://github.com/lohanidamodar/audit"
```

Comment:
Will we be merging with these sources before `utopia-php/abuse#4` and `utopia-php/audit#9` are completed?

Reply:
Yes, I think we should keep them on separate branches until the release of this branch to avoid maintaining multiple versions for them.